feature-configs/demo/performance/operator_catalogsource.patch.yaml

```diff
@@ -4,4 +4,4 @@ metadata:
   name: performance-addon-operators-catalogsource
   namespace: openshift-marketplace
 spec:
-  image: quay.io/slintes/performance-addon-operators-registry
+  image: quay.io/slintes/performance-addon-operator-registry:20200121-1756
```

Comment:
this is fine for now. I'm following up on how to handle upstream images automatically and will address this soon.